src/app/application.cpp

```diff
@@ -144,11 +143,16 @@ Application::Application(const QString &id, int &argc, char **argv)
     QPixmapCache::setCacheLimit(PIXMAP_CACHE_SIZE);
 #endif
 
-    const bool portableModeEnabled = m_commandLineArgs.profileDir.isEmpty() && QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR);
+    const bool portableModeEnabled = m_commandLineArgs.profileDir.isEmpty()
+            && QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR);
 
     const QString profileDir = portableModeEnabled
         ? QDir(QCoreApplication::applicationDirPath()).absoluteFilePath(DEFAULT_PORTABLE_MODE_PROFILE_DIR)
         : m_commandLineArgs.profileDir;
+    const QString appId = QLatin1String("qBittorrent-") + Utils::Misc::getUserIDString() + '@' + profileDir
+            + (m_commandLineArgs.configurationName.isEmpty() ? QString {} : '/' + m_commandLineArgs.configurationName);
```

Comment:
I would add parentheses to it: `('/' + m_commandLineArgs.configurationName)`

Reply:
Ok, changed.